stellar/frontend/lists-client/src/index.ts

```
@@ -103,10 +103,46 @@ export interface RegistrationInput {
   updated_ms: Option<u64>;
 }
 
-export type ContractKey = {tag: "ContractOwner", values: void} | {tag: "ListsNumber", values: void} | {tag: "Lists", values: void} | {tag: "ListAdmins", values: void} | {tag: "OwnedList", values: void} | {tag: "RegistrantList", values: void} | {tag: "RegistrationsNumber", values: void} | {tag: "Registrations", values: void} | {tag: "ListRegistration", values: void} | {tag: "RegistrationsIDs", values: void} | {tag: "Upvotes", values: void} | {tag: "UserUpvotes", values: void};
+export type ContractKey = {tag: "ContractOwner", values: void} | {tag: "ListsNumber", values: void} | {tag: "Lists", values: readonly [u128]} | {tag: "ListAdmins", values: readonly [u128]} | {tag: "OwnedList", values: readonly [string]} | {tag: "RegistrantList", values: readonly [string]} | {tag: "RegistrationsNumber", values: void} | {tag: "Registrations", values: readonly [u128]} | {tag: "ListRegistration", values: readonly [u128]} | {tag: "RegistrationsIDs", values: readonly [string]} | {tag: "Upvotes", values: readonly [u128]} | {tag: "UserUpvotes", values: readonly [string]};
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)